stable/rabbitmq/README.md

```diff
@@ -94,6 +94,7 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `ldap.tls.enabled`                           | Enable TLS for LDAP connections                  | `false` (if set to true, check advancedConfiguration parameter in values.yml)   |
 | `service.type`                               | Kubernetes Service type                          | `ClusterIP`                                             |
 | `service.port`                               | Amqp port                                        | `5672`                                                  |
+| `seervice.LoadBalancerIP`                    | LoadBalancerIP for the service                   | `nil`                                                   |
```

Comment:
`seervice.LoadBalancerIP` should be `service.loadBalancerIP` (note the removed `e` and the lowercased `l`)